src/Rpc/ErrorInfo.php

```diff
@@ -98,43 +100,44 @@ public function __construct($data = NULL) {
     }
 
     /**
-     * The type of the error. This is a constant value that identifies the
-     * proximate cause of the error. Error types are unique within a particular
-     * source of errors. This should be at most 63 characters and match
+     * The reason of the error. This is a constant value that identifies the
+     * proximate cause of the error. Error reasons are unique within a particular
+     * domain of errors. This should be at most 63 characters and match
      * /[A-Z0-9_]+/.
      *
-     * Generated from protobuf field <code>string type = 1;</code>
+     * Generated from protobuf field <code>string reason = 1;</code>
      * @return string
      */
-    public function getType()
+    public function getReason()
```

Comment:
This appears to be a BC break. I am not sure why they would change `type` to `reason` like this, but it's sure to break things if we release this PR without a major version bump.

I'd be really curious what caused this change and how it should be handled, as this is not typically something our protos do.